src/parser/user-extractor-transformer.ts

```diff
@@ -0,0 +1,68 @@
+import configuration from "../configuration/config-reader";
+import { GetActivity } from "../get-activity";
+import { GitHubIssue, GitHubIssueComment } from "../github-types";
+import { Result, Transformer } from "./processor";
+
+/**
+ * Creates entries for each bounty hunter with its associated comments.
+ */
+export class UserExtractorTransformer implements Transformer {
+  private readonly _configuration = configuration["user-extractor"];
+
+  get enabled(): boolean {
+    return true;
+  }
+
+  /**
+   * Checks if the comment is made by a human user, and if it not a command.
+   * @param comment
+   */
+  _checkEntryValidity(comment: GitHubIssueComment) {
+    return !!comment.user && !!comment.body && comment.user.type === "User";
```

Comment:
```suggestion
    return comment.body && comment.user?.type === "User";
```

Perhaps that is more concise.